builder/package.json

```diff
@@ -32,6 +34,10 @@
     "glob": "^11.0.0",
     "next": "14.2.5",
     "tsup": "^8.2.4",
-    "typescript": "^5.5.4"
+    "typescript": "^5.5.4",
+    "vitest": "^2.1.1"
```

Comment:
Could you please use `catalog:` as the version for the packages in https://github.com/flarelabs-net/poc-next/blob/main/pnpm-workspace.yaml#L5

Reply:
done! 🙂👍